Makefile

```diff
@@ -0,0 +1,32 @@
+### HGCTPGEmulatorTester_S1 Makefile
```

Comment:
Shouldn't this Makefile be in the Stage 1 subdirectory?

Reply:
the Makefile also takes care of compiling the classes from CMSSW (in `L1Trigger` directory), so it seemed to be an adequate location. This could still be changed though.

Reply:
I guess then it should also at some point compile the Stage 2 code. So that's ok if it is the intent. Otherwise the file should probably have a name that explicitly tells it is for Stage 1 only.